### PR TITLE
[DOCS] framework deprecation notice

### PR DESCRIPTION
**Pull request by kblaszczak-intel**

### CODE REVIEW

**Review by msmykx-intel — APPROVED** [review]

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 @sphinxdirective
 
+
+.. warning::
+
+   Note that OpenVINO support for Kaldi, is currently being deprecated and will be removed entirely in the future.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md:8]:
```suggestion
   Note that OpenVINO support for Kaldi is currently being deprecated and will be removed entirely in the future.
```

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 @sphinxdirective
 
+.. warning::
+
+   Note that OpenVINO support for Apache MXNet, is currently being deprecated and will be removed entirely in the future.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md:7]:
```suggestion
   Note that OpenVINO support for Apache MXNet is currently being deprecated and will be removed entirely in the future.
```

File: docs/MO_DG/prepare_model/convert_model/kaldi_specific/Aspire_Tdnn_Model.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 @sphinxdirective
 
+.. warning::
+
+   Note that OpenVINO support for Kaldi, is currently being deprecated and will be removed entirely in the future.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/kaldi_specific/Aspire_Tdnn_Model.md:7]:
```suggestion
   Note that OpenVINO support for Kaldi is currently being deprecated and will be removed entirely in the future.
```

File: docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_GluonCV_Models.md [hunk 4]
```diff
@@ -2,6 +2,11 @@
 
 @sphinxdirective
 
+.. warning::
+
+   Note that OpenVINO support for Apache MXNet, is currently being deprecated and will be removed entirely in the future.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_GluonCV_Models.md:7]:
```suggestion
   Note that OpenVINO support for Apache MXNet is currently being deprecated and will be removed entirely in the future.
```

File: docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_Style_Transfer_From_MXNet.md [hunk 5]
```diff
@@ -2,6 +2,13 @@
 
 @sphinxdirective
 
+
+.. warning::
+
+   Note that OpenVINO support for Apache MXNet, is currently being deprecated and will be removed entirely in the future.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_Style_Transfer_From_MXNet.md:8]:
```suggestion
   Note that OpenVINO support for Apache MXNet is currently being deprecated and will be removed entirely in the future.
```

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md (hunk 1 above)
Comment (sgolebiewski-intel) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md:8]:
```suggestion
   Note that OpenVINO support for Kaldi, is currently being deprecated and will be removed entirely in future releases.
```

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md (hunk 2 above)
Comment (sgolebiewski-intel) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md:7]:
```suggestion
   Note that OpenVINO support for Apache MXNet, is currently being deprecated and will be removed entirely in future releases.
```

File: docs/MO_DG/prepare_model/convert_model/kaldi_specific/Aspire_Tdnn_Model.md (hunk 3 above)
Comment (sgolebiewski-intel) [inline docs/MO_DG/prepare_model/convert_model/kaldi_specific/Aspire_Tdnn_Model.md:7]:
```suggestion
   Note that OpenVINO support for Kaldi, is currently being deprecated and will be removed entirely in future releases.
```

File: docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_GluonCV_Models.md (hunk 4 above)
Comment (sgolebiewski-intel) [inline docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_GluonCV_Models.md:7]:
```suggestion
   Note that OpenVINO support for Apache MXNet, is currently being deprecated and will be removed entirely in future releases.
```

File: docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_Style_Transfer_From_MXNet.md (hunk 5 above)
Comment (sgolebiewski-intel) [inline docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_Style_Transfer_From_MXNet.md:8]:
```suggestion
   Note that OpenVINO support for Apache MXNet, is currently being deprecated and will be removed entirely in future releases.
```

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Caffe.md [hunk 6]
```diff
@@ -2,6 +2,12 @@
 
 @sphinxdirective
 
+
+.. warning::
+
+   Note that OpenVINO support for Caffe, is currently being deprecated and will be removed entirely in the future.
```
[the remark this answers is not in the document]
Comment (kblaszczak-intel) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Caffe.md]: future releases might suggest "23.1 onwards" ... will stay with the more general wording not to introduce ambiguity.

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md (hunk 1 above)
Comment (kblaszczak-intel) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md]: future releases might suggest "23.1 onwards" ... will stay with the more general wording not to introduce ambiguity.

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md (hunk 2 above)
Comment (kblaszczak-intel) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md]: future releases might suggest "23.1 onwards" ... will stay with the more general wording not to introduce ambiguity.

File: docs/MO_DG/prepare_model/convert_model/kaldi_specific/Aspire_Tdnn_Model.md (hunk 3 above)
Comment (kblaszczak-intel) [reply docs/MO_DG/prepare_model/convert_model/kaldi_specific/Aspire_Tdnn_Model.md]: future releases might suggest "23.1 onwards" ... will stay with the more general wording not to introduce ambiguity.

File: docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_Style_Transfer_From_MXNet.md (hunk 5 above)
Comment (kblaszczak-intel) [reply docs/MO_DG/prepare_model/convert_model/mxnet_specific/Convert_Style_Transfer_From_MXNet.md]: future releases might suggest "23.1 onwards" ... will stay with the more general wording not to introduce ambiguity.

**Comment by slyalin** [comment]
Should we also update TOC by adding [DEPRECATED] to documents that are entirely dedicated for MXNet, Caffe, and Kaldi? Like `Converting an MXNet model` etc.

**Comment by kblaszczak-intel** [comment]
> Should we also update TOC by adding [DEPRECATED] to documents that are entirely dedicated for MXNet, Caffe, and Kaldi? Like `Converting an MXNet model` etc.

We sure can. I'm not sure how much we want to stress this deprecation so early in the process, so I wouldn't decide myself. But if we have more voices supporting this idea, I'm all for it.

**Comment by jane-intel** [comment]
Are we willing to offer users with mxnet, caffe or kaldi models to consider converting them to onnx? This would allow them to continue working with OpenVINO as a deployment solution.

**Comment by kblaszczak-intel** [comment]
The changes have been merged to master, so that it is easier to get feedback and potential adjustments. The feedback will be reflected in https://github.com/openvinotoolkit/openvino/pull/17537) and ported back to master.

**Pull request merged**